### ISSUE TITLE
I didn't understand coding well

### ISSUE THREAD

**Issue by Reict232** [SammyIAm/Moppy2#220]
I don’t understand which sketch needs to be loaded into the Arduino for everything to work. It’s interesting to collect, but I don’t have any knowledge. The instructions are not complete.

**Comment by samphonic** [comment]
`Moppy2-Arduino.ino` is the sketch that should be opened in the Arduino IDE.  [The wiki](https://github.com/Sammy1Am/Moppy2/wiki/Getting-Started#arduino-ide) instructions should be fairly complete.  Is there a particular step you're getting stuck on?

**Comment by Reict232** [comment]
The INO file contains plain text instead of code. I don't speak English well and the translation of the instructions is bad, if you can direct the firmware file I will be very happy :)

**Comment by Reict232** [comment]
The INO file contains plain text instead of code. I don't speak English well and the translation of the instructions is bad, if you can direct the firmware file I will be very happy :)

**Comment by samphonic** [comment]
> The INO file contains plain text instead of code.

This is correct.  The Arduino IDE will compile the code in the `src` directory when you open that sketch.  Please make sure you have unzipped the folder first, before opening it in the Arduino IDE.

I'm sorry the translation is not clear.  Would you mind explaining the steps you have completed so far?  What step are you stuck on?  What is the error/problem you are seeing?

**Comment by Reict232** [comment]
I unpacked the archive, opened the ino file, and after loading it into the Arduino, nothing worked for me. I thought it was because the file was empty and wrote a question...
I'll try to download the firmware again
 P.S. Sorry, it seems I forgot to install drivers for the Arduino copies :)

**Comment by Reict232** [comment]
Yes, when I downloaded the driver, the firmware loaded. Thanks for the support and for the cool project.
